### PR TITLE
Check Doorman log path is a valid path

### PR DESCRIPTION
**Pull request by andrewandante**
Just had a silent error here where it collapsed because it couldn't write to the path I set - so the Queue stalled at initialising.

### CODE REVIEW

**Comment by andrewandante** [comment]
Possibly needs an `is_writeable()` check as well?

File: src/Tasks/Engines/DoormanRunner.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ public function runQueue($queue)
             BASE_PATH . "/vendor/silverstripe/framework/cli-script.php dev/tasks/ProcessJobQueueChildTask"
         );
         $logPath = Environment::getEnv('SS_DOORMAN_LOGPATH');
-        if ($logPath) {
+        if ($logPath && is_dir($logPath)) {
```
Comment (emteknetnz) [inline src/Tasks/Engines/DoormanRunner.php]:
```suggestion
        if ($logPath && is_dir($logPath) && is_writable($logPath)) {
```

Comment (emteknetnz) [inline src/Tasks/Engines/DoormanRunner.php]: What happens if the dir doesn't yet exist?  It's not automatically created somewhere via `mkdir()`?

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Left a suggestion to add is_writable() as you suggested

Also a question around mkdir

Happy to merge as is after those have been answered/actioned

File: src/Tasks/Engines/DoormanRunner.php (hunk 1 above)
Comment (andrewandante) [reply src/Tasks/Engines/DoormanRunner.php]: No it's not, it's just appended to the stdOut and stdErr calls if it's set.

**Review by emteknetnz — APPROVED** [review]
Travis failures are existing

**Pull request merged**